src/acuvity/apexextend.py

```diff
@@ -333,16 +371,18 @@ def __build_scan_request(
         files: Union[Sequence[Union[str,os.PathLike]], os.PathLike, str, None] = None,
         request_type: Union[ScanrequestType,str] = ScanrequestType.INPUT,
         annotations: Optional[Dict[str, str]] = None,
-        analyzers: Optional[List[str]] = None,
-        bypass_hash: Optional[str] = None,
+        analyzers: Optional[List[str]] = [],
         anonymization: Union[ScanrequestAnonymization, str, None] = None,
-        redactions: Optional[List[str]] = None,
-        keywords: Optional[List[str]] = None,
-        access_policy: Optional[str] = None,
-        content_policy: Optional[str] = None,
+        redactions: Optional[List[str]] = [],
+        keywords: Optional[List[str]] = [],
+        guard_config: GuardConfig,
```

Comment:
only `guard_config` is an option to this function now. Please remove the rest from the function signature.